src/pyfaf/queries.py

```diff
@@ -742,7 +743,7 @@ def prioritize_longterm_problems(min_fa, problem_tuples):
         if problem.first_occurrence.day != 1:
             months -= 1
 
-        problem.rank = rank / float(months)
+        problem.rank = rank // float(months)
```

Comment:
This should stay just /

Reply:
You are right, it should stay ```/``` because the second operator is ```float```.